docs/2.8.0/docs/daml/reference/templates.rst

```diff
@@ -71,8 +71,8 @@ executed). For a contract of template ``T``, the ``self`` binding is of type
 
 .. _daml-ref-template-let:
 
-Template-local Definitions
-**************************
+Template-local Definitions (Deprecated)
```

Comment:
Hi @akrmn.  Can we add a before and after code example to make this really concrete?  I think this one will do https://daholdings.slack.com/archives/C01UC8LP76Y/p1692122312487989?thread_ts=1692115771.111369&cid=C01UC8LP76Y but leave it up to you.

Reply:
thank you @hrischuk-da that's a great example, I'll add it shortly

Reply:
done now